docs/user/ppl/cmd/addtotals.md

```diff
@@ -0,0 +1,105 @@
+# AddTotals
+
+
+## Description
+
+The `addtotals` command computes the sum of numeric fields and appends a
+row with the totals to the result. The command can also add row totals
+and add a field to store row totals. This is useful for creating summary
+reports with subtotals or grand totals. The `addtotals` command only
+sums numeric fields (integers, floats, doubles). Non-numeric fields in
+the field list are ignored even if it\'s specified in field-list or in
+the case of no field-list specified.
+
+## Syntax
+
+`addtotals [field-list] [label=<string>] [labelfield=<field>] [row=<boolean>] [col=<boolean>] [fieldname=<field>]`
+
+- `field-list`: Optional. Comma-separated list of numeric fields to sum.
+  If not specified, all numeric fields are summed.
+- `row=<boolean>`: Optional. Calculates total of each row and add a new
+  field with the total. Default is true.
+- `col=<boolean>`: Optional. Calculates total of each column and add a
+  new event at the end of all events with the total. Default is false.
+- `labelfield=<field>`: Optional. Field name to place the label. If it
+  specifies a non-existing field, adds the field and shows label at the
+  summary event row at this field. This is applicable when col=true.
+- `label=<string>`: Optional. Custom text for the totals row
+  labelfield\'s label. Default is \"Total\". This is applicable when
+  col=true. This does not have any effect when labelfield and fieldname
+  parameter both have same value.
+- `fieldname=<field>`: Optional. Calculates total of each row and add a
+  new field to store this total. This is applicable when row=true.
+
+## Example 1: Basic Example
+
+The example shows placing the label in an existing field.
+
+PPL query:
+
+    os> source=accounts | head 3|fields firstname, balance | addtotals col=true labelfield='firstname' label='Total';
+    fetched rows / total rows = 4/4
+    +-----------+---------+-------+
+    | firstname | balance | Total |
+    |-----------+---------+-------|
+    | Amber     | 39225   | 39225 |
+    | Hattie    | 5686    | 5686  |
+    | Nanette   | 32838   | 32838 |
+    | Total     | 77749   | null  |
+    +-----------+---------+-------+
+
```

Comment:
Thanks for keeping our docs fresh!  

Unfortunately since this PR was opened, all PPL docs have been migrated to Markdown. To ensure your new commands are included in doctests and formatted properly, please enclose each command in `ppl` code block and provide the expected output in a codeblock following the `ppl` block. Example: 
````
```ppl
search source=accounts | where age > 25 | fields firstname, lastname
```

Expected output:

```text
+-------------+------------+
| firstname   | lastname   |
|-------------+------------|
| Amber       | Duke       |
| Hattie      | Bond       |
+-------------+------------+
```
````

See https://github.com/opensearch-project/sql/blob/5f963a0a0ae29e20d84306d3423daf104cddeb42/docs/dev/testing-doctest.md#markdown-format-new---currently-for-docsuserppl-only